### ISSUE TITLE
Problems with visualizing results

### ISSUE THREAD

**Issue by sjc6834** [SysCV/sam-hq#78]
I tested the DIS5K dataset with train.py but the visualization is out of my expectation, can anyone help me please?
![1_0_0](https://github.com/SysCV/sam-hq/assets/130900070/9f0715e9-1a96-499a-be53-f62a081d91c7)

**Comment by lkeab** [comment]
this is usually related to the wrongly loaded checkpoint

**Comment by sweetdream33** [comment]
Hello. Have you resolved the issue? I am also experiencing the same problem.

**Comment by sjc6834** [comment]
> Hello. Have you resolved the issue? I am also experiencing the same problem.
I think caused by the prompts not fitting, when I calculate the bounding boxes as an input through masks or give the exact point prompts, it's fine

**Comment by bhack** [comment]
Yes I have the same issue. Do you think we could introduce some safety check in the bbox prompt?
I think fixing bbox it could be enough as the current eval is running only on `bboxes`:
https://github.com/SysCV/sam-hq/blob/322488826bda616798901c6280d13a9a90444ae7/train/train.py#L545
